src/common/fe.h

```diff
@@ -141,6 +141,7 @@ typedef enum {
 	FE_GUI_FOCUS,
 	FE_GUI_FLASH,
 	FE_GUI_COLOR,
+	FE_GUI_RELOADCOLORS,
```

Comment:
Just add this to the end of the enum to avoid changing the values of everything else.